jest.config.mjs

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/** @type {import('@jest/types').Config} */
+const config = {
```

Comment:
This is a stopgap solution to get `rehype-stringify` tests to work. Will expand more in the [Slack thread](https://hashicorp.slack.com/archives/C02T80JTNS2/p1682028713094729) 👍

Reply:
✅ have updated this - we no longer need separate Jest configs, this PR now uses the "dynamic config" approach from `web-platform-packages`.